pkg/config/tide.go

```diff
@@ -940,6 +959,12 @@ func (cp *TideContextPolicy) IsOptional(c string) bool {
 	if sets.New[string](cp.OptionalContexts...).Has(c) {
 		return true
 	}
+	for _, regexContext := range cp.OptionalRegexContexts {
+		re := regexp.MustCompile(regexContext)
```

Comment:
@smg247 can you follow up and change this so that we do not recompile all regexes all the time? ideally they'd be compiled once at startup and we'd store and use that

Reply:
Yeah, that's a good idea. I'll take care of that

Reply:
https://github.com/kubernetes-sigs/prow/pull/304